telco-examples/edge-clusters/edge-metallb-multi-node/bmh-node1-example.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: node1-example-credentials
+type: Opaque
+data:
+  username: ${BMC_NODE1_USERNAME}
+  password: ${BMC_NODE1_PASSWORD}
+---
+apiVersion: metal3.io/v1alpha1
+kind: BareMetalHost
+metadata:
+  name: node1-example
+  labels:
+    cluster-role: control-plane
+spec:
+  online: true
+  bootMACAddress: ${BMC_NODE1_MAC}
+  bmc:
+    #address: idrac-virtualmedia://10.168.200.32/redfish/v1/Systems/System.Embedded.1
```

Comment:
If this is used as an example, should it be in the README instead?

Reply:
yeah, makes sense to put it into the README. Let me change it